yoda/llava_data_prep/table_templates/sec_edgar/AAPL.py

```diff
@@ -0,0 +1,189 @@
+synth_tables = []
+
+# Template 1: AAPL_Revenue_simple_columns
+ex1 = {"columns": "\\begin{tabular}{| l c c |}\n\\hline\n",
```

Comment:
Could you please make these variable names more descriptive? Not sure what ext1 etc means.